ultraLatency.py

```diff
@@ -53,6 +54,11 @@
     print("Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL))
     exit(1)
 
+if options.stepSize:
```

Comment:
There is no reason for this check if you have a default value. What is it you are trying to check here anyway?